test/dlc_tests/sanity/test_telemetry.py

```diff
@@ -289,11 +331,23 @@ def import_framework(image_uri, container_name, docker_cmd, framework, job_type,
         framework_to_import = framework.replace("huggingface_", "")
         framework_to_import = "torch" if framework_to_import == "pytorch" else framework_to_import
         ec2_connection.run(f"{docker_cmd} run --name {container_name} -id {image_uri} bash")
-        output = ec2_connection.run(
-            f"{docker_cmd} exec -i {container_name} python -c 'import {framework_to_import}; import time; time.sleep(5)'",
-            warn=True
-        )
-        assert output.ok, f"'import {framework_to_import}' fails when credentials not configured"
+        if test_mode:
+            ec2_connection.run(
+                f"{docker_cmd} exec -i -e TEST_MODE={test_mode} {container_name} python -c 'import mxnet; import time; time.sleep(5);'",
+                warn=True
+            )
+            output = ec2_connection.run(
+                f"{docker_cmd} exec -i {container_name} /bin/bash -c 'cat /tmp/test_request.txt'",
+                warn=True
+            ).stdout.strip('\n')
+        else:
+            output = ec2_connection.run(
+                f"{docker_cmd} exec -i {container_name} python -c 'import {framework_to_import}; import time; time.sleep(5)'",
+                warn=True
```

Comment:
why set warn to True here and handle error as assertion error?

Reply:
This is a copy paste of other code mentioned here https://github.com/aws/deep-learning-containers/blob/master/test/dlc_tests/sanity/test_telemetry.py#L294, so thought to keep it same.